cmd/maya-apiserver/cstor-operator/cspc/handler.go

```diff
@@ -137,6 +143,29 @@ func (c *Controller) syncCSPC(cspc *apis.CStorPoolCluster) error {
 		return nil
 	}
 
+	if !cspc.DeletionTimestamp.IsZero() {
```

Comment:
it would be good if we can make this as a fn rather than in syncCSPC

Reply:
Done